EnvironmentVariables.md

```diff
@@ -0,0 +1,60 @@
+# Configuration Via Environment Variables
+
+Many features in an Engine Yard Cloud stable-v6 environment can now be enabled
+and configured via [environment variables](https://support.cloud.engineyard.com/hc/en-us/articles/360007661794).
+
+This makes working with the Engine Yard Cloud platform much easier and 
+removes the need for custom Chef recipes in many cases.
+
+A list of supported settings is documented in the next section.
+If you are missing a setting, please request it by [opening a GitHub issue](https://github.com/engineyard/ey-cookbooks-stable-v7/issues/new).
+
+## Supported Settings
```

Comment:
@rupikakapoor we are missing some settings here, particularly the ones about versions of PostgreSQL.
Also, existing ones like EY_SIDEKIQ_* are pointing to empty files.

Reply:
yeah, this will be addresses in FBZ-10526